### DISCUSSION TITLE
Shortcut CTRL + UP/DOWN

### DISCUSSION THREAD

**Discussion by SarenT** [keepassxreboot/keepassxc#10190] [Q&A]
Keepass does something (as if there is a modification) on the database when pressed CTRL + UP/DOWN on Ubuntu. It is unclear what is happening and the docs do not mention anything about this combination. Also the CTRL + /  does not do anything on 2.7.6. Does anyone know, what this key combination is doing?

**Comment by stdedos** [comment]
`CTRL + /` works in Win10 + 

```
KeePassXC - Έκδοση 2.7.6
Αναθεώρηση: dd21def
```

... but true, neither `KeePassXC_KeyboardShortcuts.html` nor https://keepassxc.org/docs/KeePassXC_UserGuide#_keyboard_shortcuts mention anything about `CTRL + UP/DOWN`.

Maybe ... they are sorting the folders' view? 😅

**Reply by SarenT** [reply]
Ah I see but no it did not sort anything :)

**Reply by droidmonkey** [reply]
When a group is selected, CTRL+Up/Down sorts the child groups by A-Z or Z-A respectively. This should be added to the keyboard shortcuts list. There also seems to be a bug since this shouldn't happen unless the group view is in focus.
